students/WesleyM/Lesson4/mailroom2.py

```diff
@@ -0,0 +1,55 @@
+donors = [
+  {'name': 'Alice Adams Ron', 'donation': [20]} ,
+  {'name': 'Bob Be-Lake', 'donation': [100]},
+  {'name': 'Charles Cruz', 'donation': [30, 50, 10]},
+  {'name': 'Denise Dnice', 'donation': [10, 5]},
+  {'name': 'Edward Eduardo', 'donation': [25, 20, 20, 20, 10]}
+  ]
+ 
+
+def send_ty():
+  donor = input("Enter a donor name or input 'List' for a list of donors\n>")
+  if donor.lower() == "list":
+    for names in donors:
+        print(names['name'])
+    use_input()
+  else:
+    donation = float(input(("Enter a donation amount for {} \n>").format(donor)))
+    for names in donors:
+      if donor == names['name']:
+        names['donation'].append(donation)
+        break
+    else:
+      donors.append({'name': donor, 'donation': [donation]})
+    print("Thank you {} for your donation of ${:.2f}".format(donor, donation))
+  use_input()
+  
+def letters():
+  for name in donors:
+    print('Printing Letter to {a}'.format(a = name['name']))
+    f = open('{a}.txt'.format(a = name['name']), 'w')
+    f.write('Dear Ms./Mrs./Mr./Dr. {x}, \n We are thankful for your donation of ${y}. Your donation will be used for (insert harmful activity here). We hope you donate again soon!'.format(x = name['name'], y = sum(name['donation'])))
+    f.close()
```

Comment:
Remember you can use triple quotes to preserve returns and indents in your strings and stay under the Pep8 line character limit.